--- gpac-0.5.0-svn5277/applications/mp4box/main.c.0008~	2014-06-15 18:41:46.311455692 +0200
+++ gpac-0.5.0-svn5277/applications/mp4box/main.c	2014-06-15 18:42:47.252029539 +0200
@@ -1404,7 +1404,7 @@ GF_DashSegmenterInput *set_dash_input(GF
 				if (strlen(opts+7) > 99) {
 					GF_LOG(GF_LOG_ERROR, GF_LOG_DASH, ("[DASH] PeriodID cannot exceed 99 characters in MP4Box, truncating ...\n"));
 				}
-				strncpy(di->periodID, opts+7, 99);
+				strncpy(di->periodID, opts+7, 199);
 			}
 			else if (!strnicmp(opts, "bandwidth=", 10)) di->bandwidth = atoi(opts+10);
 			else if (!strnicmp(opts, "role=", 5)) strncpy(di->role, opts+5, 99);
