--- gpac-0.5.0-svn5277/configure.0012~	2014-06-15 19:07:21.999298520 +0200
+++ gpac-0.5.0-svn5277/configure	2014-06-15 19:07:22.022632074 +0200
@@ -101,6 +101,7 @@ has_xmlrpc="no"
 has_openjpeg="no"
 gprof_build="no"
 static_build="no"
+static_libs="no"
 want_pic="no"
 has_joystick="no"
 has_xul="no"
@@ -230,6 +231,7 @@ GPAC configuration options:
   --disable-platinum       disable Platinum UPnP support
   --disable-alsa           disable Alsa audio
   --disable-oss-audio      disable OSS audio
+  --disable-sdl            disable SDL support
   --enable-jack            enable  Jack audio
   --disable-jack           disable Jack audio
   --enable-pulseaudio      enable  Pulse audio
@@ -245,6 +247,7 @@ GPAC configuration options:
   --enable-amr-wb          enable AMR WB library
   --enable-amr             enable both AMR NB and WB libraries
   --enable-static-bin      GPAC static build
+  --enable-static-lib      GPAC static libraries build
   --static-mp4box          configure for static linking of MP4Box.
   --enable-depth           enables depth handling in the compositor
 
@@ -1774,6 +1777,8 @@ for opt do
             ;;
         --enable-static-bin) static_build="yes";
             ;;
+        --enable-static-lib) static_libs="yes";
+            ;;
         --disable-ipv6) has_ipv6="no"
             ;;
         --disable-wx) has_wx="no"
@@ -1782,6 +1787,8 @@ for opt do
             ;;
         --disable-oss-audio) has_oss_audio="no"
             ;;
+        --disable-sdl) has_sdl="no"
+            ;;
         --disable-x11-shm) has_x11_shm="no"
             ;;
         --disable-x11-xv) has_x11_xv="no"
@@ -3053,6 +3060,7 @@ echo "CONFIG_AMR_WB_FT=$has_amr_wb" >> c
 echo "DEBUGBUILD=$debuginfo" >> config.mak
 echo "GPROFBUILD=$gprof_build" >> config.mak
 echo "STATICBUILD=$static_build" >> config.mak
+echo "STATICLIBS=$static_libs" >> config.mak
 
 echo "CONFIG_IPV6=$has_ipv6" >> config.mak
 if test "$has_ipv6" = "yes" ; then
--- gpac-0.5.0-svn5277/Makefile.0012~	2014-06-15 19:07:22.005965251 +0200
+++ gpac-0.5.0-svn5277/Makefile	2014-06-15 19:07:22.025965439 +0200
@@ -139,9 +139,6 @@ else
 	$(INSTALL) $(INSTFLAGS) -m 755 bin/gcc/libgpac.$(DYN_LIB_SUFFIX).$(VERSION_SONAME) $(DESTDIR)$(prefix)/$(libdir)/libgpac.$(DYN_LIB_SUFFIX).$(VERSION_SONAME)
 	ln -sf libgpac.$(DYN_LIB_SUFFIX).$(VERSION_SONAME) $(DESTDIR)$(prefix)/$(libdir)/libgpac.so.$(VERSION_MAJOR)
 	ln -sf libgpac.$(DYN_LIB_SUFFIX).$(VERSION_SONAME) $(DESTDIR)$(prefix)/$(libdir)/libgpac.so
-ifeq ($(DESTDIR)$(prefix),$(prefix))
-	ldconfig || true
-endif
 endif
 endif
 
@@ -158,7 +155,9 @@ ifeq ($(GPAC_ENST), yes)
 	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/include/gpac/enst/*.h "$(DESTDIR)$(prefix)/include/gpac/enst"
 endif
 	mkdir -p "$(DESTDIR)$(prefix)/$(libdir)"
+ifeq ($(STATICLIBS),yes)
 	$(INSTALL) $(INSTFLAGS) -m 644 "./bin/gcc/libgpac_static.a" "$(DESTDIR)$(prefix)/$(libdir)"
+endif
 	$(MAKE) installdylib
 
 uninstall-lib:
--- gpac-0.5.0-svn5277/src/Makefile.0012~	2014-06-15 19:07:22.002631885 +0200
+++ gpac-0.5.0-svn5277/src/Makefile	2014-06-15 19:09:05.553684116 +0200
@@ -607,12 +607,16 @@ compositor: $(LIBGPAC_COMPOSITOR)
 	@echo "OBJS $(OBJS)"
 	@echo "LIBS $(EXTRALIBS)"
 ifeq ($(CONFIG_DARWIN),yes)
+ifeq ($(STATICLIBS),yes)
 	$(LIBTOOL) -s -o ../bin/gcc/libgpac_static.a $(OBJS)
 	$(RANLIB) ../bin/gcc/libgpac_static.a
+endif
 	$(CC) $(SHFLAGS) $(LD_SONAME) $(LDFLAGS) -o $@ $(OBJS) $(EXTRALIBS) 
 else
+ifeq ($(STATICLIBS),yes)
 	$(AR) cr ../bin/gcc/libgpac_static.a $(OBJS)
 	$(RANLIB) ../bin/gcc/libgpac_static.a
+endif
 	$(CC) $(SHFLAGS) $(LD_SONAME) $(LDFLAGS) -o $@ $(OBJS) $(EXTRALIBS)
 ifeq (,$(findstring yes, $(CONFIG_WIN32)))
 	mv $@ $@.$(VERSION_SONAME)
