diff -rupN gpac-26.02.0.old/applications/gpac/Makefile gpac-26.02.0/applications/gpac/Makefile
--- gpac-26.02.0.old/applications/gpac/Makefile	2026-02-05 14:26:28.000000000 +0100
+++ gpac-26.02.0/applications/gpac/Makefile	2026-04-06 15:48:10.875927424 +0200
@@ -58,8 +58,6 @@ LINKFLAGS+=-lgpac
 ifeq ($(CONFIG_EMSCRIPTEN),yes)
 else ifeq ($(CONFIG_DARWIN),yes)
 #LINKFLAGS+= -Wl,-rpath,'@loader_path'
-else
-LINKFLAGS+= -Wl,-rpath,'$$ORIGIN' -Wl,-rpath,$(prefix)/lib -Wl,-rpath-link,../../bin/gcc
 endif
 endif
 
diff -rupN gpac-26.02.0.old/applications/mp4box/Makefile gpac-26.02.0/applications/mp4box/Makefile
--- gpac-26.02.0.old/applications/mp4box/Makefile	2026-02-05 14:26:28.000000000 +0100
+++ gpac-26.02.0/applications/mp4box/Makefile	2026-04-06 15:48:36.772879754 +0200
@@ -23,8 +23,6 @@ LINKFLAGS=-L../../bin/gcc -L../../extra_
 ifeq ($(CONFIG_EMSCRIPTEN),yes)
 else ifeq ($(CONFIG_DARWIN),yes)
 #LINKFLAGS+= -Wl,-rpath,'@loader_path'
-else
-LINKFLAGS+= -Wl,-rpath,'$$ORIGIN' -Wl,-rpath,$(prefix)/lib -Wl,-rpath-link,../../bin/gcc
 endif
 
 ifeq ($(STATIC_BUILD),yes)
diff -rupN gpac-26.02.0.old/modules/common.mak gpac-26.02.0/modules/common.mak
--- gpac-26.02.0.old/modules/common.mak	2026-02-05 14:26:28.000000000 +0100
+++ gpac-26.02.0/modules/common.mak	2026-04-06 15:48:59.063598298 +0200
@@ -11,6 +11,4 @@ endif
 
 ifeq ($(CONFIG_EMSCRIPTEN),yes)
 else ifeq ($(CONFIG_DARWIN),yes)
-else
-LDFLAGS+= -Wl,-rpath,'$$ORIGIN' -Wl,-rpath,$(prefix)/lib -Wl,-rpath-link,../../bin/gcc
 endif
