--- gpac-0.5.0-svn5277/configure~	2014-06-15 21:39:27.260242158 +0200
+++ gpac-0.5.0-svn5277/configure	2014-06-15 21:49:53.316173020 +0200
@@ -1647,6 +1647,8 @@ if $cc -o $TMPE $TMPC $LDFLAGS > /dev/nu
     has_xmlrpc="yes"
 fi
 
+xml2_cflags="`$pkg_config --cflags libxml-2.0`"
+xml2_libs="`$pkg_config --libs libxml-2.0`"
 
 
 #look for alsa
@@ -3117,6 +3119,8 @@ if test "$has_dvb4linux" = "yes"; then
 fi
 
 echo "XMLRPC_INC=$has_xmlrpc" >> config.mak
+echo "XML2_CFLAGS=$xml2_cflags" >> config.mak
+echo "XML2_LIBS=$xml2_libs" >> config.mak
 
 if test "$has_oss_audio" != "no"; then
     echo "OSS_INC_TYPE=$has_oss_audio" >> config.mak
