--- gpac-0.5.0-svn5619/applications/osmo4_wx/wxGPACControl.cpp.omv~	2015-05-06 18:47:52.188200526 +0200
+++ gpac-0.5.0-svn5619/applications/osmo4_wx/wxGPACControl.cpp	2015-05-06 18:47:59.098268048 +0200
@@ -398,7 +398,7 @@ wxGPACControl::wxGPACControl(wxWindow *p
 
 	/*audio dec enum*/
 	sOpt = gf_cfg_get_key(cfg, "Systems", "DefAudioDec");
-	u32 count = gf_modules_get_count(m_pApp->m_user.modules);
+	/*u32*/ count = gf_modules_get_count(m_pApp->m_user.modules);
 	GF_BaseDecoder *ifc_d;
 	select = 0;
 	s32 to_sel = 0;
