--- gpac-0.5.0.svn5178/Makefile.0015~	2014-06-15 17:23:26.576666322 +0200
+++ gpac-0.5.0.svn5178/Makefile	2014-06-15 17:23:26.579999687 +0200
@@ -78,6 +78,9 @@ endif
 endif
 ifeq ($(DISABLE_ISOFF), no)
 	$(INSTALL) $(INSTFLAGS) -m 755 bin/gcc/MP4Box "$(DESTDIR)$(prefix)/bin"
+ifeq ($(DISABLE_M2TS), no)
+	$(INSTALL) $(INSTFLAGS) -m 755 bin/gcc/mp42ts "$(DESTDIR)$(prefix)/bin"
+endif
 endif
 ifeq ($(DISABLE_PLAYER), no)
 	$(INSTALL) $(INSTFLAGS) -m 755 bin/gcc/MP4Client "$(DESTDIR)$(prefix)/bin"
