--- gpac-0.5.0.svn5178/modules/directfb_out/Makefile.directfb~	2014-06-15 17:13:13.420874343 +0200
+++ gpac-0.5.0.svn5178/modules/directfb_out/Makefile	2014-06-15 17:13:41.227804630 +0200
@@ -31,7 +31,7 @@ all: $(LIB)
 
 
 $(LIB): $(OBJS)
-	$(CC) $(SHFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(LDFLAGS) -L../../bin/gcc -lgpac
+	$(CC) $(SHFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(LDFLAGS) -L../../bin/gcc -lgpac $(DIRECTFB_LIB)
 
 clean: 
 	rm -f $(OBJS) ../../bin/gcc/$(LIB)
