--- gpac-0.5.0.svn5178/modules/jack/Makefile.0016~	2014-03-22 18:32:48.147082370 +0100
+++ gpac-0.5.0.svn5178/modules/jack/Makefile	2014-06-15 17:17:42.473420855 +0200
@@ -25,7 +25,7 @@ LIB=gm_jack.$(DYN_LIB_SUFFIX)
 all: $(LIB)
 
 $(LIB): $(OBJS)
-	$(CC) $(SHFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac -L/usr/lib  -ljack
+	$(CC) $(SHFLAGS) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac -L/usr/lib  -ljack
 
 clean: 
 	rm -f $(OBJS) ../../bin/gcc/$(LIB)
--- gpac-0.5.0.svn5178/modules/oss_audio/Makefile.0016~	2014-03-22 18:32:43.346142390 +0100
+++ gpac-0.5.0.svn5178/modules/oss_audio/Makefile	2014-06-15 17:17:42.473420855 +0200
@@ -35,7 +35,7 @@ LIB=gm_oss_audio.$(DYN_LIB_SUFFIX)
 all: $(LIB)
 
 $(LIB): $(OBJS)
-	$(CC) $(SHFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac $(OSS_LDFLAGS)
+	$(CC) $(SHFLAGS) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac $(OSS_LDFLAGS)
 ifeq ($(STATICBUILD),yes)
 	$(CC) $(SHFLAGS) $(LDFLAGS) -o ../../bin/gcc/gm_oss_audio-static.$(DYN_LIB_SUFFIX) $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac_static $(OSS_LDFLAGS)
 endif
--- gpac-0.5.0.svn5178/modules/pulseaudio/Makefile.0016~	2014-03-22 18:32:44.197131751 +0100
+++ gpac-0.5.0.svn5178/modules/pulseaudio/Makefile	2014-06-15 17:17:42.473420855 +0200
@@ -25,7 +25,7 @@ LIB=gm_pulseaudio.$(DYN_LIB_SUFFIX)
 all: $(LIB)
 
 $(LIB): $(OBJS)
-	$(CC) $(SHFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac -L/usr/lib -lpulse-simple
+	$(CC) $(SHFLAGS) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac -L/usr/lib -lpulse-simple -lpulse
 
 clean: 
 	rm -f $(OBJS) ../../bin/gcc/$(LIB)
