diff -up gpac-1.0.1/src/utils/os_file.c.omv~ gpac-1.0.1/src/utils/os_file.c
--- gpac-1.0.1/src/utils/os_file.c.omv~	2021-09-01 01:40:50.133983792 +0200
+++ gpac-1.0.1/src/utils/os_file.c	2021-09-01 01:40:58.849033643 +0200
@@ -52,6 +52,7 @@
 
 #endif
 
+#undef printf
 
 GF_EXPORT
 GF_Err gf_rmdir(const char *DirPathName)
diff -up gpac-1.0.1/src/utils/zutil.h.omv~ gpac-1.0.1/src/utils/zutil.h
--- gpac-1.0.1/src/utils/zutil.h.omv~	2021-09-01 01:41:21.050160667 +0200
+++ gpac-1.0.1/src/utils/zutil.h	2021-09-01 01:50:06.614925243 +0200
@@ -48,6 +48,8 @@ extern int errno;
 #endif
 /* compile with -Dlocal if your debugger can't find static symbols */
 
+#define FAR /* T */
+
 typedef unsigned char  uch;
 typedef uch FAR uchf;
 typedef unsigned short ush;
@@ -225,21 +227,9 @@ typedef int ptrdiff_t;
 #if defined(STDC) && !defined(HAVE_MEMCPY) && !defined(NO_MEMCPY)
 #  define HAVE_MEMCPY
 #endif
-#ifdef HAVE_MEMCPY
-#  ifdef SMALL_MEDIUM /* MSDOS small or medium model */
-#    define zmemcpy _fmemcpy
-#    define zmemcmp _fmemcmp
-#    define zmemzero(dest, len) _fmemset(dest, 0, len)
-#  else
-#    define zmemcpy memcpy
-#    define zmemcmp memcmp
-#    define zmemzero(dest, len) memset(dest, 0, len)
-#  endif
-#else
-extern void zmemcpy  OF((Bytef* dest, const Bytef* source, uInt len));
-extern int  zmemcmp  OF((const Bytef* s1, const Bytef* s2, uInt len));
-extern void zmemzero OF((Bytef* dest, uInt len));
-#endif
+#define zmemcpy memcpy
+#define zmemcmp memcmp
+#define zmemzero(dest, len) memset(dest, 0, len)
 
 /* Diagnostic functions */
 #ifdef DEBUG
diff -up gpac-1.0.1/applications/generators/SVG/Makefile.omv~ gpac-1.0.1/applications/generators/SVG/Makefile
--- gpac-1.0.1/applications/generators/SVG/Makefile.omv~	2021-09-01 01:59:58.476529617 +0200
+++ gpac-1.0.1/applications/generators/SVG/Makefile	2021-09-01 02:02:01.649121292 +0200
@@ -2,7 +2,7 @@ include ../../../config.mak
 
 vpath %.c $(SRC_PATH)/applications/generators/SVG
 
-CFLAGS= $(OPTFLAGS) -I"$(SRC_PATH)/include"
+CFLAGS= $(OPTFLAGS) -I"$(SRC_PATH)/include" $(shell pkg-config --cflags libxml-2.0)
 
 ifeq ($(DEBUGBUILD),yes)
 CFLAGS+=-g
@@ -18,7 +18,7 @@ endif
 OBJS= html.o laser.o main.o v1.o v2.o v3.o
 
 CFLAGS+=-g
-LDFLAGS+=-g
+LDFLAGS+=-g $(shell pkg-config --libs libxml-2.0)
 CFLAGS+=$(XML2_CFLAGS)
 
 ifeq ($(CONFIG_WIN32),yes)
